src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import {string, node, oneOf} from 'prop-types';
+import style from './Thumbnail.st.css';
+import Check from 'wix-ui-icons-common/Check';
+import {Thumbnail as CoreThumbnail, ThumbnailProps as CoreThumbnailProps} from 'wix-ui-core/Thumbnail';
+import {withStylable} from 'wix-ui-core/withStylable';
+import {Text} from '../Text';
+
+export interface ThumbnailProps extends CoreThumbnailProps {
+  /** image of the thumbnail */
+  image?: React.ReactElement<any>;
+  /** title of the thumbnail */
+  title: string;
+  /** description of the thumbnail */
+  description?: string;
+}
+
+const StyledThumbnail = withStylable<CoreThumbnailProps>(
+  CoreThumbnail,
+  style,
+  () => ({})
+);
+
+export const Thumbnail: React.SFC<ThumbnailProps> = props => {
+  const {children, title, image, description, ...rest} = props;
+
+  return (
+      <StyledThumbnail {...rest} selectedIcon={<Check size="24"/>}>
```

Comment:
you can also set the size using css.
let's talk about it and understand the prefered way.

Reply:
Are we sure we want to move to this pattern and deprecate the withStylable func. I will feel more comfortable discussing it with Barak first because there might be something that we are missing that led to this hoc func.
In any case we have this in use in a lot of components, so they need to be refactored as well